### ISSUE TITLE
Error 500 - Internal Server Error, site_creator service not running

### ISSUE THREAD

**Issue by nartya** [frappe/frappe_docker#393]
Hi, I got a slightly different problem. I followed all the instructions but when I run " docker-compose up -d ", I don't see the frappe_docker_site-creator_1 service running and on the web I got a 500 error code. When I run " docker logs frappe_docker_site-creator_1 -f " I got the following lines in my screenshot. I don't know what I should modify in the docker instance.
![okok](https://user-images.githubusercontent.com/35570611/103160211-4ae29680-47d2-11eb-8582-890db36b9194.PNG)

**Comment by revant** [comment]
By looking at the last line I think something to do with docker networking? I don't know exactly. I'm out of ideas.

**Comment by nartya** [comment]
Me too. I got those errors in the screenshot above when I run " docker-compose up -d " for the first time. When I run it again, It only tries to restart site-creator service, It says it's done but the service stops automatically then when I check again the logs I got this. My site already exists but I still got an internal error in my browser.
![Capture](https://user-images.githubusercontent.com/35570611/103216904-79b65500-4917-11eb-83db-510aac5d34cf.PNG)
